aptos-node/src/storage.rs

```diff
@@ -13,12 +12,39 @@ use aptos_config::{
 use aptos_db::AptosDB;
 use aptos_executor::db_bootstrapper::maybe_bootstrap;
 use aptos_logger::{debug, info};
-use aptos_storage_interface::DbReaderWriter;
+use aptos_storage_interface::{DbReader, DbReaderWriter};
 use aptos_types::waypoint::Waypoint;
 use aptos_vm::AptosVM;
-use std::{fs, path::Path, sync::Arc, time::Instant};
+use std::{fs, net::SocketAddr, path::Path, sync::Arc, time::Instant};
 use tokio::runtime::Runtime;
 
+#[cfg(not(feature = "consensus-only-perf-test"))]
+pub(crate) fn bootstrap_db(
+    aptos_db: AptosDB,
+    backup_service_address: SocketAddr,
+) -> (Arc<AptosDB>, DbReaderWriter, Option<Runtime>) {
+    use aptos_backup_service::start_backup_service;
+
+    let (aptos_db, db_rw) = DbReaderWriter::wrap(aptos_db);
+    let db_backup_service = start_backup_service(backup_service_address, aptos_db.clone());
+    (aptos_db, db_rw, Some(db_backup_service))
+}
+
+#[cfg(feature = "consensus-only-perf-test")]
+pub(crate) fn bootstrap_db(
```

Comment:
This one is missing a comment. Since FakeAptosDB is already well-documented, maybe just something simple like "Use an in-memory DB (FakeAptosDB) and avoid writing db backups."?